### PR TITLE
Instructions

### PR DESCRIPTION
**Pull request by markumina**
to build..

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+int main()
+{
+   printf("\nit has to build.\n\n");
```
Comment (kalmanchapman) [inline main.c]: nice!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # codener-chat
 there are worse things
+
+## build
+gcc -Wall -Wextra main.c -o whatever & ./whatever
```
Comment (mnkhouri) [inline README.md]: ¯\\\_(ツ)\_/¯

Comment (mnkhouri) [inline README.md]:
```suggestion
gcc -Wall -Wextra main.c -o whatever && ./whatever
```

File: main.c (hunk 1 above)
Comment (markumina) [reply main.c]: ok...

**Pull request merged**